File: 2024/day1/star1.py
id1 = []
id2 = []
sum = 0
for line in open("2024/day1/input.txt", "r"):
    ltokens = line.split()
    id1.append(int(ltokens[0]))
    id2.append(int(ltokens[1]))

for a,b in zip(id1, id2):
    sum+=abs(a-b)

print(sum)

File: 2022/day4/star1.cpp
#include <fstream>
#include <string>
#include <iostream>
#include <sstream>
int main() 
{
    std::ifstream input("input.txt");
    std::string line;
    int sum = 0, a, b, x, y;
    while(getline(input, line)) 
    {
        std::stringstream ss(line);
        char c;
        ss >> a >> c >> b >> c >> x >> c >> y;

        sum+=(a<=x && b>=y) || (x<=a && y>=b);

    }
    std::cout << sum << std::endl;
}